common/autoware_auto_perception_rviz_plugin/include/autoware_auto_perception_rviz_plugin/object_detection/object_polygon_display_base.hpp

```diff
@@ -153,12 +261,91 @@ class AUTOWARE_AUTO_PERCEPTION_RVIZ_PLUGIN_PUBLIC ObjectPolygonDisplayBase
     m_marker_common.load(config);
   }
 
-  void update(float wall_dt, float ros_dt) override { m_marker_common.update(wall_dt, ros_dt); }
+  void update(float wall_dt, float ros_dt) override
+  {
+    m_marker_common.update(wall_dt, ros_dt);
+    point_cloud_common->update(wall_dt, ros_dt);
+  }
 
   void reset() override
   {
+    static int reset_count = 0;
+    RCLCPP_INFO(
+      rclcpp::get_logger("autoware_auto_perception_plugin"), "reset called %d times",
+      reset_count++);
+
     RosTopicDisplay::reset();
     m_marker_common.clearMarkers();
+    point_cloud_common->reset();
+  }
+
+  void subscribe()
+  {
+    if (!RosTopicDisplay::isEnabled()) {
+      return;
+    }
+
+    RosTopicDisplay::subscribe();
+    points_subscribe();
+  }
+
+  void updateTopic() override
+  {
+    static int update_count = 0;
+    RCLCPP_INFO(
+      rclcpp::get_logger("autoware_auto_perception_plugin"), "updateTopic called %d times",
+      update_count++);
+    unsubscribe();
+    reset();
+    subscribe();
+    this->context_->queueRender();
+  }
+
+  void points_subscribe()
+  {
+    if (!m_publish_objs_pointcloud->getBool()) {
+      return;
+    }
+
+    if (m_default_pointcloud_topic->isEmpty()) {
+      RosTopicDisplay::setStatus(
+        rviz_common::properties::StatusProperty::Error, "Topic",
+        QString("Error subscribing: Empty topic name"));
+      return;
+    }
+
+    try {
+      rclcpp::SubscriptionOptions sub_opts;
+      sub_opts.event_callbacks.message_lost_callback = [&](rclcpp::QOSMessageLostInfo & info) {
+        std::ostringstream sstm;
```

Comment:
[WANT] Please fix the spell check CI is failed:
`sstream` seems reasonable for me